### PR TITLE
Introduces simple messagebox

### PR DESCRIPTION
**Pull request by terw-dan**
I've created a very simple messagebox for testing https://github.com/Yoast/wordpress-seo/issues/6297
This probably needs to be improved on styling and UI/UX.

### CODE REVIEW

**Comment by CarolineGeven** [comment]
@hedgefield and @moorscode Could you please advise UX wise? 
When the onboarding wizard cannot be run, the following message will be shown:
schermafbeelding 2017-05-30 om 11 42 25

Please advise UX wise. I think the box itself would have to move down to be correctly aligned with the normal UX:
schermafbeelding 2017-05-30 om 11 42 37

But perhaps there are other things that we need to consider

**Comment by hedgefield** [comment]
The message box itself looks fine to me (there is a little more padding on the bottom than on the top it looks like, that should be corrected), but mainly there should be space between the logo and the box yeah. Try 40px.

**Comment by CarolineGeven** [comment]
@andizer Can you implement the suggestions @hedgefield gave?

File: composites/OnboardingWizard/MessageBox.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import YoastLogo from "../basic/YoastLogo";
+
+import interpolateComponents from "interpolate-components";
+
+/**
+ * Creates a message box in the style of the onboarding wizard.
+ *
+ * @param {object} props Object containing the text and link to render in the messagebox.
+ *
+ * @returns {React.element} The rendered messagebox.
```
Comment (IreneStr) [inline composites/OnboardingWizard/MessageBox.js]: {ReactElement} The rendered message box.

File: composites/OnboardingWizard/MessageBox.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ */
+const MessageBox = ( props ) => (
+	<div className="yoast-wizard-body">
+		<YoastLogo height={93} width={200}/>
```
Comment (IreneStr) [inline composites/OnboardingWizard/MessageBox.js]: Please add spaces

**Comment by IreneStr** [comment]
Acceptance done 👍

**Pull request merged**